v22.2/drop-owned-by.md

```diff
@@ -0,0 +1,38 @@
+---
+title: DROP OWNED BY
+summary: The DROP OWNED BY statement drops all objects owned by and any grants on objects not owned by a role.
+toc: true
+docs_area: reference.sql
+---
+
+The `DROP OWNED BY` [statement](sql-statements.html) drops all objects owned by and any grants on objects not owned by a role.
+
+{% include {{ page.version.version }}/misc/schema-change-stmt-note.md %}
+
+## Required privileges
+
+The role must have the `DROP` [privilege](security-reference/authorization.html#managing-privileges) on the specified objects. `DROP OWNED BY` cannot be performed if the
+user has synthetic privileges.
+
+## Synopsis
+
+<div>{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/drop_owned_by.html %}</div>
+
+## Parameters
+
+ Parameter | Description
+-----------|------------
+`role_spec_list` | The source role, or a comma-separated list of source roles.
+`RESTRICT` | _(Default)_ Do not drop ownership if any objects (such as [constraints](constraints.html) and tables) use it.
+`CASCADE` | Drop other objects that depend on the objects being dropped.<br><br>`CASCADE` does not list objects it drops, so should be used cautiously.
```

Comment:
looks like `DROP OWNED BY ... CASCADE` is not implemented yet : https://github.com/cockroachdb/cockroach/blob/87ed064dc23eab6948ee8a07e8507f150bda0e44/pkg/sql/schemachanger/scbuild/internal/scbuildstmt/drop_owned_by.go#L81-L81